SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaCommandlet.cpp

```diff
@@ -17,10 +18,16 @@ int32 UGenerateSchemaCommandlet::Main(const FString& Args)
 	UE_LOG(LogSpatialGDKEditorCommandlet, Display, TEXT("Schema Generation Commandlet Started"));
 
 	//NOTE: For future use, if schema generation configuration at the command line is desired
-	//TArray<FString> Tokens;
-	//TArray<FString> Switches;
-	//TMap<FString, FString> Params;
-	//ParseCommandLine(*Args, Tokens, Switches, Params);
+	TArray<FString> Tokens;
+	TArray<FString> Switches;
+	TMap<FString, FString> Params;
+	ParseCommandLine(*Args, Tokens, Switches);
+
+	if (Switches.Contains("delete-schema-db"))
```

Comment:
Can we move this option to a common location?

Reply:
Not a bad idea, but would require some docs refactor. Could you raise a ticket to investigate?
In the meantime, what about having the SchemaAndSnapshots commandlet extend from this one? Would then be trivial to reuse functionality.